Funcky/Extensions/EnumerableExtensions/Intersperse.cs

```diff
@@ -7,6 +7,6 @@ public static partial class EnumerableExtensions
     public static IEnumerable<TSource> Intersperse<TSource>(this IEnumerable<TSource> source, TSource element)
         => source.WithFirst()
             .SelectMany(s => s.IsFirst
-                ? Sequence.Return(s.Value)
-                : Sequence.Return(element, s.Value));
+                ? Funcky.Sequence.Return(s.Value)
```

Comment:
That does not look good. 
* Why is this ambigous? Is Sequence now ambigious everywhere?
* the extension Sequence should not interfere witht he class Sequence.

Reply:
Its because `Sequence` is a member of the `EnumerableExtensions` class